### PR TITLE
[ADD] 12.0 Sale Tier Validation module added

### PR DESCRIPTION
**Pull request by mgosai**
**Added module:**
sale_tier_validation

### CODE REVIEW

**Comment by dreispt** [comment]
The PR author belongs to Open Source Integrators, I believe we have an organization CLA.

File: sale_tier_validation/README.rst
Comment (nikul-serpentcs) [inline sale_tier_validation/README.rst]: 12.0

**Review by nikul-serpentcs — COMMENTED** [review]
README generator Base on https://github.com/OCA/maintainer-tools#readme-generator

File: sale_tier_validation/README.rst
Comment (mgosai) [reply sale_tier_validation/README.rst]: @nikul-serpentcs : addressed!

**Review by dreispt — APPROVED** [review]

**Comment by rousseldenis** [comment]
@mgosai Could you add 'server-ux' in oca_dependencies ?

**Comment by dreispt** [comment]
@rousseldenis great point! I wonder why the build is green despite that.
@mgosai Can you do the honors please?

**Comment by mgosai** [comment]
> @mgosai Could you add 'server-ux' in oca_dependencies ?

Request Addressed and updated.
Thanks

**Comment by rousseldenis** [comment]
> @mgosai Could you add 'server-ux' in oca_dependencies ?

@mgosai I would have added '.txt' :smile: Not in the module but at the root of repository. Thanks

**Comment by mgosai** [comment]
> > @mgosai Could you add 'server-ux' in oca_dependencies ?
> 
> @mgosai I would have added '.txt'  Not in the module but at the root of repository. Thanks

lol, done. Thanks

**Comment by dreispt** [comment]
@rousseldenis Looks good now?

**Comment by rousseldenis** [comment]
Could you squash your commits before merge ?

**Review by AaronHForgeFlow — COMMENTED** [review]
I'd be great to include at least one test

**Comment by dreispt** [comment]
@AaronHForgeFlow Checking the module and it doesn't include a single line of business logic. So, IMO, in this case a test would bring little value.

**Comment by LoisRForgeFlow** [comment]
I agree there is no need to add tests, as logic is tested in base module. However, squashing your commits is still pending and I thinks that's what is blocking the merge. Could you please address that?

Thanks for contributing.

**Review by ps-tubtim — APPROVED** [review]
Code LGTM, Please squash commits

**Pull request merged**
